pkg/consoleui/openshift.go

```diff
@@ -14,8 +14,8 @@ const (
 	openShiftConsoleNS                = "openshift-console"
 	openShiftConsoleRouteName         = "console"
 	openShiftPipelineNamespaceViewURL = "https://%s/pipelines/ns/%s/pipeline-runs"
-	openShiftPipelineDetailViewURL    = "https://%s/k8s/ns/%s/tekton.dev~v1beta1~PipelineRun/%s"
-	openShiftPipelineTaskLogURL       = "%s/logs/%s"
+	openShiftPipelineDetailViewURL    = "https://%s/k8s/ns/%s/tekton.dev~v1~PipelineRun/%s"
+	openShiftPipelineTaskLogURL       = "%s/logs?taskName=%s"
```

Comment:
Shall we need to update the doc as well with this changed query `taskName`

Reply:
i am going to revert that change.. we would only this one when version below 4.15 openshift will be out of support